crates/sui-indexer-alt-graphql/src/api/types/validator_set.rs

```diff
@@ -24,55 +20,19 @@ use crate::{
 pub(crate) type CValidator = JsonCursor<u64>;
 
 /// Representation of `0x3::validator_set::ValidatorSet`.
-#[derive(Clone, Debug, SimpleObject)]
-#[graphql(complex)]
+#[derive(Clone, Debug)]
 pub(crate) struct ValidatorSet {
-    #[graphql(skip)]
     scope: Scope,
-
-    #[graphql(skip)]
-    active_validators: Vec<ValidatorV1>,
-
-    #[graphql(skip)]
-    at_risk_validators: VecMap<NativeSuiAddress, u64>,
-
-    /// Total amount of stake for all active validators at the beginning of the epoch.
-    pub total_stake: Option<BigInt>,
-
-    /// Validators that are pending removal from the active validator set, expressed as indices in
-    /// to `activeValidators`.
-    pub pending_removals: Option<Vec<u64>>,
-
-    // TODO: instead of returning the id and size of the table, potentially return the table itself, paginated.
-    /// Object ID of the wrapped object `TableVec` storing the pending active validators.
-    pub pending_active_validators_id: Option<SuiAddress>,
-
-    /// Size of the pending active validators table.
-    pub pending_active_validators_size: Option<u64>,
-
-    /// Object ID of the `Table` storing the mapping from staking pool ids to the addresses
-    /// of the corresponding validators. This is needed because a validator's address
-    /// can potentially change but the object ID of its pool will not.
-    pub staking_pool_mappings_id: Option<SuiAddress>,
-
-    /// Size of the stake pool mappings `Table`.
-    pub staking_pool_mappings_size: Option<u64>,
-
-    /// Object ID of the `Table` storing the inactive staking pools.
-    pub inactive_pools_id: Option<SuiAddress>,
-
-    /// Size of the inactive pools `Table`.
-    pub inactive_pools_size: Option<u64>,
-
-    /// Object ID of the `Table` storing the validator candidates.
-    pub validator_candidates_id: Option<SuiAddress>,
-
-    /// Size of the validator candidates `Table`.
-    pub validator_candidates_size: Option<u64>,
+    native: ValidatorSetV1,
 }
 
-#[ComplexObject]
+#[Object]
 impl ValidatorSet {
+    /// Total amount of stake for all active validators at the beginning of the epoch.
```

Comment:
nit: function definition order (to make it easier to find the function in the `impl` block).